evaluator/autoware_planning_evaluator/src/metrics/deviation_metrics.cpp

```diff
@@ -45,6 +45,28 @@ Accumulator<double> calcLateralDeviation(const Trajectory & ref, const Trajector
   return stat;
 }
 
+Accumulator<double> calcNearestPoseDeviation(
+  const Trajectory & prev_traj, const Trajectory & ref_traj, const Pose & ego_pose)
+{
+  Accumulator<double> stat;
+
+  if (prev_traj.points.empty() || prev_traj.points.empty()) {
+    return stat;
+  }
+
+  const auto prev_idx =
+    autoware::motion_utils::findNearestIndex(prev_traj.points, ego_pose.position);
+  const auto prev_p = prev_traj.points[prev_idx];
+  const auto ref_idx = autoware::motion_utils::findNearestIndex(ref_traj.points, ego_pose.position);
+  const auto ref_p = ref_traj.points[ref_idx];
+
+  const double nearest_dist = autoware::universe_utils::calcDistance2d(prev_p, ref_p);
```

Comment:
better to use `calcLateralDeviation` .

if using `calcDistance2d` and interval of points of two trajectories is diffrent, the distance also become large.

Reply:
`calcLateralOffset` is also ok

Reply:
Yes sir!
update to use `calcLateralOffset`